content/icerpc/connection/security-with-tls.md

```diff
@@ -86,13 +86,13 @@ await using var connection = new ClientConnection(
 ```
 
 The ssl transport is provided solely for backward compatibility with Ice: the standard way for an Ice application to
-request a secure connection is to send a proxy with an ssl server address. When IceRPC + Slice receives such a proxy,
-the ssl transport captures this information ("TLS required") and ensures the client establishes a secure connection when
-it calls the decoded proxy.
+request a secure connection is to use a proxy with an ssl server address. When IceRPC + Slice decodes a service
```

Comment:
More of a random thought, but should we be capitalizing `ssl` as `SSL` everywhere since it is an acronym?

Reply:
It's because ssl (lowercase) is the transport name we use in service addresses (proxy strings in Ice). Same for tcp, quic, ws, wss, bt ... they are all lowercase.